### ISSUE TITLE
If resized folders exist (but are empty) there is no resizing (i.e., they are not populated)

### ISSUE THREAD

**Issue by ebgoldstein** [Doodleverse/segmentation_gym#97]
deep in the issue https://github.com/Doodleverse/segmentation_gym/issues/92 thread, there seems to be a little hiccup in make_datasets when the `resized` folders (`resized_images` and `resized_labels`):

if these directories exist, but are empty (or not filled with the correct images), then the whole resized operation is skipped.

https://github.com/Doodleverse/segmentation_gym/blob/1ccde41c24a0b76c1e2bcb7a96de1ccea4c60c77/make_nd_dataset.py#L236-L262

So, Q for @dbuscombe-usgs :
- is this something to fix via code (search for each file and make sure it exists? Or just remake resized images?)
- is this something to fix via documentation (tell people to make sure to delete resized image folders before running make_datasets?)
- is there another option?

**Comment by ebgoldstein** [comment]
actually i am reading down the code and it may already be implemented in some form?

**Comment by dbuscombe-usgs** [comment]
The skipping of resized folder generation serves the needs of those who use the same dataset to make multiple sets of npzs, for example for creating 2-, 3- and 4-class datasets

Yes you are correct that if the prescribed workflow is not followed, or if something goes wrong, the directories 'resized_images' and 'resized_labels' would need to be deleted (or whatever it takes) 

`is this something to fix via documentation (tell people to make sure to delete resized image folders before running make_datasets?)`

No, ordinarily resized images and folders directories should not be deleted, if created properly


`is this something to fix via code (search for each file and make sure it exists? Or just remake resized images?)`

Yes, this would be the solution. Program could be updated to

1. check directories are not empty
2. check length of files in both directories are the same

anything else to check?

`actually i am reading down the code and it may already be implemented in some form?`

Specifically, what may already be implemented?

**Comment by dbuscombe-usgs** [comment]
Honestly I dont really know under what cirumstances the folders would be empty. Seems like a non-issue

**Comment by ebgoldstein** [comment]
yep, agreed.. lets close it.. woudl only be empty if make_datasets fails...
